scripts/actions/mobskills/exuviation.lua

```diff
@@ -14,16 +16,57 @@ end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local effectCount = 0
-    local dispel = mob:eraseStatusEffect()
 
-    while (dispel ~= xi.effect.NONE)
-    do
-        effectCount = effectCount + 1
-        dispel = mob:eraseStatusEffect()
+    -- Use the same debuff list as Taxet.lua for consistency
+    local removables = {
```

Comment:
Even though it'll be long, can you make this a vertical table with a single entry per line? I know there are other places in the codebase that aren't laid out nicely, but those are outliers